src/qt/walletmodel.cpp

```diff
@@ -523,7 +522,7 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
     }
 
     // Short-circuit if we are returning a bumped transaction PSBT to clipboard
-    if (create_psbt) {
+    if (retval == QMessageBox::Save) {
```

Comment:
As `m_psbt_button = button(QMessageBox::Save);` is not near, maybe add a comment which describes that this code actually handles "Create Unsigned" button press?

Is `// Short-circuit if we are returning a bumped transaction PSBT to clipboard` comment above still relevant?